### PR TITLE
Refactoring for same url state, moved to Siddhi 4.0.0-M7

### PR DESCRIPTION
**Pull request by Yasara123**
Tests OSGI module is commented until feature release.

### CODE REVIEW

File: tests/osgi-tests/pom.xml [hunk 1]
```diff
@@ -167,6 +167,24 @@
             <type>zip</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.ops4j.pax.exam</groupId>
+            <artifactId>pax-exam-junit4</artifactId>
+            <version>3.6.0</version>
```
Comment (suhothayan) [inline tests/osgi-tests/pom.xml]: Please move all dependencies to parent poms add the versions in properties section and then refer them here.

Comment (Yasara123) [inline tests/osgi-tests/pom.xml]: Fixed in 7a6cb32

File: pom.xml [hunk 2]
```diff
@@ -194,6 +193,24 @@
...
+            <dependency>
+                <groupId>org.ops4j.pax.exam</groupId>
+                <artifactId>pax-exam-spi</artifactId>
+                <version>4.10.0</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.ops4j.pax.exam</groupId>
+                <artifactId>pax-exam-spi</artifactId>
+                <version>4.10.0</version>
```
Comment (suhothayan) [inline pom.xml]: You have to also move them to the properties section

Comment (Yasara123) [inline pom.xml]: Fixed in  	4de7975

**Pull request merged**